namespace-overview/design.md

```diff
@@ -1,21 +1,18 @@
 # Namespace Overview
```

Comment:
The view shows Filter and Sort components.

What are available sorting fields ?  Name and Status?
Are we allowing them to filter by anything else?

Reply:
We could sort by number of services as well